### PR TITLE
Feat/acf support

### PR DESCRIPTION
**Pull request by toutma**
Adds ACF logic -> Twig Extensions

### CODE REVIEW

File: src/Resources/config/services.yml [hunk 1]
```diff
@@ -1,13 +1,7 @@
...
-    wordpress.acf.cache_dir: "%kernel.cache_dir%/wp/acf"
-    wordpress.acf.debug: "%kernel.debug%"
-    wordpress.acf.export_json: "%kernel.root_dir%/../data/acf/acf-export/acf-export.json"
-
-    wordpress.menus.class: EasyWordpressBundle\Service\WordpressMenus
-
 services:
   wordpress.twig_extension:
     class: EasyWordpressBundle\Twig\Extension\WordpressTwigExtension
+    arguments: [ '@doctrine.orm.entity_manager']
```
Comment (zalexki) [inline src/Resources/config/services.yml]: Wrong spacing :
`arguments: [ '@doctrine.orm.entity_manager']` => `arguments: ['@doctrine.orm.entity_manager']`

File: src/Resources/config/services.yml [hunk 2]
```diff
@@ -1,13 +1,7 @@
-parameters:
-    wordpress.acf.cache_dir: "%kernel.cache_dir%/wp/acf"
-    wordpress.acf.debug: "%kernel.debug%"
-    wordpress.acf.export_json: "%kernel.root_dir%/../data/acf/acf-export/acf-export.json"
-
-    wordpress.menus.class: EasyWordpressBundle\Service\WordpressMenus
-
 services:
   wordpress.twig_extension:
```
Comment (zalexki) [inline src/Resources/config/services.yml]: Wrong indent maybe we can add our default .editorconfig file

File: src/Twig/Extension/WordpressTwigExtension.php [hunk 3]
```diff
@@ -2,16 +2,51 @@
...
     {
         return [
-//            new \Twig_SimpleFilter('date', array($this, 'dateFilter')),
+            new \Twig_SimpleFilter('date', array($this, 'dateFilter')),
+            new \Twig_SimpleFilter('do_shortcode', [$this, 'doShortcode']),
         ];
     }
 
+    /**
+     * @param $content
```
Comment (zalexki) [inline src/Twig/Extension/WordpressTwigExtension.php]: missing type

File: src/Twig/Extension/WordpressTwigExtension.php [hunk 4]
```diff
@@ -86,15 +121,125 @@ function (
...
+            new \Twig_SimpleFunction(
+                'wp_get_permalink', function ($id, $contentType = 'page') {
+                return get_permalink(apply_filters('wpml_object_id', $id, $contentType));
+            }
+            ),
         ];
     }
 
+    /**
+     * @param $str
```
Comment (zalexki) [inline src/Twig/Extension/WordpressTwigExtension.php]: missing type

File: src/Twig/Extension/WordpressTwigExtension.php [hunk 5]
```diff
@@ -86,15 +121,125 @@ function (
                 bloginfo($show);
             }
             ),
+
+            # ACF Get Fields
+            new \Twig_SimpleFunction(
```
Comment (zalexki) [inline src/Twig/Extension/WordpressTwigExtension.php]: The indent of these twig function seem really odd try to do a php cs fixer with PSR-2 see what happend

File: README.md [hunk 6]
```diff
@@ -85,4 +85,41 @@ symfony('wordpress.helper')->boot();
 
 Enable the bundle `new EasyWordpressBundle\EasyWordpressBundle(),` in your `AppKernel.php`
 
+
+## 3 - Features
```
Comment (zalexki) [inline README.md]: Complete documentation before merging

File: README.md [hunk 7]
```diff
@@ -1,34 +1,57 @@
...
 
-Then just `composer require extremesensio/easywordpressbundle`
+Then just `composer require extremesensio/easywordpressbundle 1.*`
 
 # 2 - Installation & usage
 ## 2.1 With Commands
-Use wp-cli to install wordpress in `web/wp` : `wp-ci core download --path=project/web/wp`
+Enable the bundle `new EasyWordpressBundle\EasyWordpressBundle(),` in your `AppKernel.php`.
+
+Use wp-cli to install wordpress in `web/wp` : `wp-ci core download --path=project/web/wp`.<br>
```
Comment (zalexki) [inline README.md]: `<br>` ?

File: README.md [hunk 8]
```diff
@@ -1,34 +1,57 @@
...
+ - create `app/config/parameters.php` file
+ - create `.env` file at root directory
+ 
 
 For every template you can use `easywordpress:generate-templates` to generate wp themes file using this annotation (do not generate twig file):
-```
-/**
-  * @TemplateName(name="Accueil")
-*/
+```php
```
Comment (zalexki) [inline README.md]: nice !

File: README.md [hunk 9]
```diff
@@ -83,6 +106,106 @@ symfony('wordpress.helper')->boot();
...
+```
+Creates nav from a WordPress menu created from the theme option Appearance.<br>
+Only `themeLocation` and `menu` variables are required.<br>
+More informations: https://developer.wordpress.org/reference/functions/wp_nav_menu/
+
+#### _e
+```twig
+  _e(text, domain)
+```
+Display translated translated text.<br>
```
Comment (zalexki) [inline README.md]: `<br>` ?

File: README.md [hunk 10]
```diff
@@ -83,6 +106,106 @@ symfony('wordpress.helper')->boot();
...
+More informations: https://developer.wordpress.org/reference/functions/_e/
+
+### __
+```twig
+  __(text, domain)
+```
+Retrieves the translation of text.<br>
```
Comment (zalexki) [inline README.md]: `<br>`?

File: README.md [hunk 11]
```diff
@@ -83,6 +106,106 @@ symfony('wordpress.helper')->boot();
...
+More informations: https://developer.wordpress.org/reference/functions/__/
+
+#### bloginfo
+```twig
+  bloginfo(info)
+```
+Displays information about the current site.<br>
```
Comment (zalexki) [inline README.md]: `<br>` ?

File: src/Resources/config/services.yml [hunk 12]
```diff
@@ -1,35 +1,29 @@
...
-    class: EasyWordpressBundle\EventListener\ControllerArgumentsSubscriber
-    arguments: [ '@wordpress.helper' ]
-    tags:
-      - { name: kernel.event_subscriber }
-
-  #wordpress.custom.post.type:
-  #  class: EasyWordpressBundle\Service\WordpressCustomPostType
+    wordpress.twig_extension:
+        class: EasyWordpressBundle\Twig\Extension\WordpressTwigExtension
+        arguments: [ '@doctrine.orm.entity_manager']
```
Comment (zalexki) [inline src/Resources/config/services.yml]: carefull spacing

File: src/Resources/config/services.yml [hunk 13]
```diff
@@ -1,35 +1,29 @@
...
+        tags:
+            - { name: twig.extension }
+
+    wordpress.helper:
+        class: EasyWordpressBundle\Service\WordpressHelper
+        arguments: [ '@kernel' ]
```
Comment (zalexki) [inline src/Resources/config/services.yml]: carefull spacing `['@kernel']`

File: src/Resources/config/services.yml [hunk 14]
```diff
@@ -1,35 +1,29 @@
...
+
+    wordpress.nav.menus:
+        class: EasyWordpressBundle\Service\WordpressNav
+
+    wordpress.data_collector:
+        class: EasyWordpressBundle\Service\WordpressDataCollector
+        arguments: []
```
Comment (zalexki) [inline src/Resources/config/services.yml]: delete it if not needed

File: src/Resources/config/services.yml [hunk 15]
```diff
@@ -1,35 +1,29 @@
...
+        tags:
+            - { name: data_collector, template: '@EasyWordpressBundle/Resources/views/toolbar.html.twig', id: 'wordpress'}
+
+    wordpress.controller_arguments_subscriber:
+        class: EasyWordpressBundle\EventListener\ControllerArgumentsSubscriber
+        arguments: [ '@wordpress.helper' ]
```
Comment (zalexki) [inline src/Resources/config/services.yml]: carefull spacing

File: src/Twig/Extension/WordpressTwigExtension.php [hunk 16]
```diff
@@ -2,55 +2,90 @@
...
+
+    /**
+     * @return array
+     */
     public function getFilters()
     {
         return [
-//            new \Twig_SimpleFilter('date', array($this, 'dateFilter')),
+            new \Twig_SimpleFilter('date', array($this, 'dateFilter')),
+            new \Twig_SimpleFilter('do_shortcode', [$this, 'doShortcode']),
```
Comment (zalexki) [inline src/Twig/Extension/WordpressTwigExtension.php]: try to be consistent `array()` or `[]`

File: src/Resources/config/services.yml (hunk 15 above)
Comment (toutma) [inline src/Resources/config/services.yml]: Service used in `WordpressHelper`

File: README.md (hunk 7 above)
Comment (toutma) [inline README.md]: Yep ;)

File: README.md (hunk 8 above)
Comment (toutma) [inline README.md]: ;)

**Pull request merged**